gemelli/preprocessing.py

```diff
@@ -152,18 +127,11 @@ def rclr_matrix(M):
 
     Examples
     --------
-
-    To use directly with a
-    prebuilt tensor.
-
-    >>> from numpy import np
-    >>> T_counts = np.array([[10, 14, 43],
-                             [ 41, 43, 14]])
-    >>> rclr_matrix(T_counts)
+    TODO
```

Comment:
There are a few instances, like here, where an example has gotten replaces with `TODO`, what is the reason for this?

Reply:
The examples need updating due to the code restructuring. Was planning to do that with tutorials etc.. in the next PR.